### PR TITLE
Fix some issues reported by the linter

### PR DESCRIPTION
**Pull request by armfazh**
No changes in the functionality.

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
         "module": "ES2020",
         "lib": [
             "ES2020",
-            "webworker"
+            "dom"
```
Comment (lukevalenta) [inline tsconfig.json:7]: What's the reason for this change? Is the library interacting with any `dom` elements?

File: src/bignum/big.ts [hunk 2]
```diff
@@ -140,19 +140,19 @@ export async function hashNums(nums: bigint[]): Promise<bigint> {
         enc = new TextEncoder()
     let totBytes = 0
     for (let i = 0; i < numNums; i++) {
-        const numBytes = enc.encode(nums[i].toString())
+        const numBytes = enc.encode(nums[i as number].toString())
```
Comment (lukevalenta) [inline src/bignum/big.ts:143]: I don't have a strong preference, but we could specify the type of `i` in the loop definition instead for a less verbose option.
```
for (let i: number = 0; i < numNums; i++) {
...
```

**Review by lukevalenta — APPROVED** [review]
I left a couple non-blocking comments, but looks good overall.

File: src/bignum/big.ts (hunk 2 above)
Comment (armfazh) [reply src/bignum/big.ts]: the linter cannot infer this from the code. So need to explicitly indicate it.
but for reading, I used the `at(i)` method.
